### PR TITLE
added requirements levels for the spec

### LINKED ISSUE #9: Need to describe explicitly minimum specification
Nowhere in the specification is the minimum representation of tracks described.

Need to specify the **minimum** representation of this specification:
{objects + links} `(MUST)`
{+ tracks} `(MAY)`

### PR DESCRIPTION
**Pull request by pcmasuzzo**
This PR aims to fix issue #9 (need to describe requirements levels for the spec).
Also - replaced one final reference to `segments `into `links` (into `workflow` image).

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ developed by the
 
 The current specification is based on *tracking by detection* techniques (see computational workflow in the following figure) and the data produced therein:
 
-![Workflow](images/Workflow.png){:class="img-responsive"}
+![workflow](images/workflow.png){:class="img-responsive"}
```
Comment (gsergeant) [inline index.md:14]: I'm assuming the capitalized image name will have no consequense with the lowercase path here? 🙂

Comment (pcmasuzzo) [reply index.md:14]: Nope. Rendering seems okay in the `Files Changed` view. Not OK for you?

Comment (gsergeant) [reply index.md:14]: Nono, looks fine. Was just wondering

**Comment by sbesson** [comment]
NB: this branch has conflicts with `origin/master`, you need to rebase the commits or merge `origin/master` into it.

File: index.md (hunk 1 above)
Comment (sbesson) [reply index.md:14]: An now the image seems to be broken in https://cellmigstandorg.github.io/Tracks/. One last rendering fix PR is required.

**Pull request merged**